### DISCUSSION TITLE
Tiffen Black Satin?

### DISCUSSION THREAD

**Discussion by horintel** [videovillage/community#159] [General]
I’d love to use Scatter, but our current diffusion for the projects are Black Satins, 1-3.
https://tiffen.com/products/black-satin-screw-in-filter

Any chance these will be added to the product?

**Comment by gregcotten** [comment]
We only profiled the most popular filters available in camera houses in Los Angeles - we did not profile Black Satin, but might consider doing a more "niche" diffusion filter profiling in the future.

**Reply by horintel** [reply]
The Black Satins and black glimmerglasses are relatively new, and have a really nice look. Definitely check the Black Satin 1,2, and 3 out.

**Reply by shaheedmalik** [reply]
Please consider Black Satin 1/8, 1/4, and 1/2 as well.

**Comment by maxjgoldberg12** [comment]
I'd strongly encourage you all to profile Black Satin, Black Glimmerglass, and Black Pearlescent. These filters have been out for many years and are quite popular.
